QModPluginEmulator/QModPluginGenerator.cs

```diff
@@ -88,30 +89,26 @@ public static void Finish()
             "PerformGarbage", "Fallback handler could not load"
         };
 
-        public static List<string> DirtyMidStrings = new List<string>()
+        public static List<string> DirtyPatterns = new List<string>()
         {
-            "\n(Filename", 
+            @"[\r\n]+(\(Filename: .*\))"
         };
 
         private static void LibcHelper_Format_Postfix(ref string __result)
         {
-            foreach(string dirtyString in DirtyStartStrings)
+            foreach (string dirtyString in DirtyStartStrings)
             {
-                if(__result.StartsWith(dirtyString))
+                if (__result.StartsWith(dirtyString))
                 {
-                    __result = "";
+                    __result = string.Empty;
                     return;
                 }
             }
 
-            foreach(string dirtyString in DirtyMidStrings)
+            foreach (string dirtyPattern in DirtyPatterns)
             {
-                int i = __result.IndexOf(dirtyString);
-                if(i >= 0)
-                {
-                    __result = __result.Remove(i);
-                    return;
-                }
+                var regex = new Regex(dirtyPattern);
```

Comment:
Strongly recommending against using `new Regex` inside a loop.  
Set up the regex instances you need ahead of time, into `static` members, and using `RegexOptions.Compiled` if possible.

Reply:
Fair point, I'll set this up

Reply:
d4990ba